README.md

```diff
@@ -1,50 +1,275 @@
 # Music Library
 
-XXXXX
+A music library API created following a CRUD REST API technology using Node.js, Express, Mocha-Chai, Postman and pgAdmin. 
```

Comment:
Nice README! Lots of detail, well laid out, very good! Only thing missing here is the setup, it would be worth describing how to spin up the database, how did you get this running? Was it a docker container? If there are any commands needed to do this they should be listed here in a setup section